app/models/conservation_status.rb

```diff
@@ -124,6 +124,8 @@ def normalize_geoprivacy
 
   def update_observation_geoprivacies
     return true if skip_update_observation_geoprivacies
+    # Don't reassess obs for newly created or destroyed non-obscuring global statuses
+    return true if ( saved_change_to_id? || destroyed? ) && geoprivacy == Observation::OPEN && place_id.nil?
```

Comment:
On staging there are 55991 ConservationStatus records where the `geoprivacy` column is `null` not `open`, so I think you're going to want to check for that here.